lib/cocoapods/target/build_settings.rb

```diff
@@ -814,12 +838,12 @@ def pod_target_xcconfig_values_by_consumer_by_key
 
         # @return [Array<Sandbox::FileAccessor>]
         define_build_settings_method :file_accessors, :memoized => true do
-          target.file_accessors.select { |fa| fa.spec.test_specification? == test_xcconfig? }
+          target.file_accessors.select { |c| c.spec.spec_type == @xcconfig_spec_type }
```

Comment:
why rename `fa` to `c`?

Reply:
I think this had a couple revs on it and just happened to land there, i'll change back